src/compile/selection/interval.ts

```diff
@@ -118,14 +118,22 @@ const interval:SelectionCompiler = {
       name: undefined,
       type: 'rect',
       encode: {
-        enter: {fill: {value: '#eee'}},
+        enter: {
```

Comment:
I guess we should make brush style configurable (in a later PR) ?  

We can just output some role name and use the role config. -- Given that we will output config in the Vega output soon in https://github.com/vega/vega-lite/pull/2372 -- This should be easily to implement.  

File an issue?